src/clients/discord.ts

```diff
@@ -92,18 +93,44 @@ function registerDiscord(): void
             bridge.twitchMessageSearchCache[msg] = listNode;
 
         let currIndex: number = 0;
-        const recursiveSend = (): void =>
+        function recursiveSend(chunkedTwitchMessages: string[], reply?: { userState: PrivateMessage; }): void
         {
             if(currIndex < chunkedTwitchMessages.length)
-                bridge.twitch.authChatClient!.say(configFile.T2D_CHANNELS[0], chunkedTwitchMessages[currIndex]).then(() =>
-                {
-                    currIndex++;
-                    //tmijs does this in their own version of sending multiple messages... therefore we must also follow this jank method
-                    setTimeout(() => recursiveSend(), 250);
-                });
-        };
-
-        recursiveSend();
+                reply?.userState ?
+                    bridge.twitch.authChatClient?.say(configFile.T2D_CHANNELS[0], chunkedTwitchMessages[currIndex], {
+                        replyTo: reply.userState
+                    }).then((): void =>
+                    {
+                        currIndex++;
+                        setTimeout(() => recursiveSend(chunkedTwitchMessages, reply ?? undefined), 250);
```

Comment:
Let's try to avoid duplicating the same arrow function!

Reply:
Also, do you think we still need the delay anymore? I imagine it's a twitch limitation, but still curious

Reply:
Fixed the arrow function, no clue about the delay; you're the one who added it.